src/CMakeLists.txt

```diff
@@ -6,7 +6,11 @@ include(ECMEnableSanitizers)
 
 set(synclib_NAME ${APPLICATION_EXECUTABLE}sync)
 
-find_package(Qt5 5.12 COMPONENTS Core Network Xml Concurrent WebEngineWidgets WebEngine REQUIRED)
+find_package(Qt5 5.12 COMPONENTS Core Network Xml Concurrent REQUIRED)
+if(WITH_WEBENGINE)
+  find_package(Qt5 5.12 COMPONENTS WebEngineWidgets WebEngine REQUIRED)
```

Comment:
I would have just made them OPTIONAL dependencies and then build can use the found variable (`Qt5WebEngine_FOUND`)
I get that we have both WebEngine and WebEngineWidgets module but adding one more option feels strange for me if teh goal is to downgrade the dependency requirement from mandatory to optional

Reply:
Makes sense. Done.